scipy/stats/tests/test_continuous_basic.py

```diff
@@ -45,9 +45,9 @@
 ]
 
 
-distslow = ['kstwo', 'ksone', 'kappa4', 'gausshyper', 'recipinvgauss',
-            'genexpon', 'vonmises', 'vonmises_line', 'cosine', 'invweibull',
-            'powerlognorm', 'johnsonsu', 'kstwobign']
+distslow = ['kstwo', 'genexpon', 'ksone', 'recipinvgauss', 'vonmises',
+            'kappa4', 'vonmises_line', 'gausshyper', 'norminvgauss',
+            'geninvgauss']
```

Comment:
I removed a few items because on my machine:
* `cosine`, `powerlognorm` and `johnsonsu` take less than 0.5 seconds, so they are not slower than `ncx2`, which isn't on this list.
* `invweibull`, and `kstwobign` are either not slow or are skipped in some other way; they don't appear on the list of the 100 slowest tests even after removing them from this list.

I added `norminvgauss` and `geninvgauss` because they took longer than `0.75s`, which I chose as a threshold.